docs/src/app/components/pages/customization/themes.md

```diff
@@ -18,10 +18,14 @@ import MuiThemeProvider from 'material-ui/lib/MuiThemeProvider';
 import getMuiTheme from 'material-ui/lib/styles/getMuiTheme';
 import AppBar from 'material-ui/lib/app-bar';
 
+// This replaces all text colors with cyan500
+// http://www.material-ui.com/#/customization/colors
 const muiTheme = getMuiTheme({palette: {textColor: Colors.cyan500}});
 
 class Main extends React.Component {
   render() {
+    // MuiThemeProvider takes the theme as a property and passed it down the hierarchy
+    // using React's context feature.
```

Comment:
@mbrookes

> Is this effectively replacing the default theme that is already context?

the default theme will be used if there are no themes on the context. Hmm i think it's a good idea to mention it :+1: